mean/modules/survey/views/p17_30.html

```diff
@@ -0,0 +1,391 @@
+                            <input type="text" class="form-control" id="hernia_location">
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            Bowel Sounds Absent
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="bowelsound" />
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            &nbsp &nbsp &nbsp &nbsp Location:
+                          </td>
+                          <td class="col-md-9">
+                            <input type="text" class="form-control" id="bowelsound_location">
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            Pain to Palpation
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="palpation" />
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            &nbsp &nbsp &nbsp &nbsp Location:
+                          </td>
+                          <td class="col-md-9">
+                            <input type="text" class="form-control" id="rales_location">
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            Masses
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="masses" />
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-2">
+                            Colostomy
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="colostomy" />
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            Ileostomy
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="ileostomy" />
+                          </td>
+                      </tr>
+                      <tr>
+                          <td class="col-md-1">
+                            PEG Tube
+                          </td>
+                          <td class="col-md-9">
+                            <input type="checkbox" name="pegtube" />
+                          </td>
+                      </tr>
+                    </table>
+    <!-- Insert pagination here -->
```

Comment:
This was carried over from the example page that I used to generate. I don't know what it is for but I left it as a reference for whoever was using it.